atlp-rwanda/atlp-pulse-fn
# Ft team details

# PR Description

team details implementation

### Description of tasks that were expected to be completed

-  coordinator and manager are able to see  team details

### How to test this PR?

- login as a` manager` or a `coordinator `
- please make sure that you have members in the team and also ratings to have a clear view of all corresponding components
- navigate to teams in the sidebar or dashboard for the manager
- click on team card and you will be able to see all details about the team
